source/languages/en/riak/dev/taste-of-riak/querying-golang.md

```diff
@@ -0,0 +1,562 @@
+---
+title: "Taste of Riak: Querying with Go"
+project: riak
+version: 1.3.1+
```

Comment:
I don't think anything should list a version prior to 1.4... but yes, this should be 1.3 compatible.